addons/mail/static/src/components/notification_list/tests/notification_list_notification_group_tests.js

```diff
@@ -98,53 +99,41 @@ QUnit.test('notification group basic layout', async function (assert) {
 });
 
 QUnit.test('mark as read', async function (assert) {
-    assert.expect(6);
+    assert.expect(3);
 
     // message that is expected to have a failure
     this.data['mail.message'].records.push({
-        id: 11, // random unique id, will be used to link failure to message
-        message_type: 'email', // message must be email (goal of the test)
-        model: 'mail.channel', // expected value to link message to channel
-        res_id: 31, // id of a random channel
-        res_model_name: "Channel", // random res model name, will be asserted in the test
+        author_id: this.data.currentPartnerId,
+        id: 11,
+        message_type: 'email',
+        model: 'mail.channel',
+        res_id: 31,
     });
     // failure that is expected to be used in the test
     this.data['mail.notification'].records.push({
         mail_message_id: 11, // id of the related message
         notification_status: 'exception', // necessary value to have a failure
-        notification_type: 'email', // expected failure type for email message
+        notification_type: 'email',
     });
-    const bus = new Bus();
-    bus.on('do-action', null, payload => {
-        assert.step('do_action');
-        assert.strictEqual(
-            payload.action,
-            'mail.mail_resend_cancel_action',
-            "action should be the one to cancel email"
-        );
-        assert.strictEqual(
-            payload.options.additional_context.default_model,
-            'mail.channel',
-            "action should have the group model as default_model"
-        );
-        assert.strictEqual(
-            payload.options.additional_context.unread_counter,
-            1,
-            "action should have the group notification length as unread_counter"
-        );
-    });
-    const { createNotificationListComponent } = await this.start({ env: { bus } });
+    const { afterNextRender, createNotificationListComponent } = await this.start();
     await createNotificationListComponent();
     assert.containsOnce(
         document.body,
         '.o_NotificationGroup_markAsRead',
         "should have 1 mark as read button"
     );
-
-    document.querySelector('.o_NotificationGroup_markAsRead').click();
-    assert.verifySteps(
-        ['do_action'],
-        "should do an action to display the cancel email dialog"
+    assert.containsOnce(
+        document.body,
+        '.o_NotificationGroup',
+        "should have a notification group"
+    );
+    await afterNextRender(() => {
```

Comment:
I meant to remove the empty line after the click, to keep the related asserts close together. But it's recommended to keep an empty line before the click (or any action).

Reply:
Ok, I got it. Thanks